symbolic_trace/opcode_translator/executor/variables/base.py

```diff
@@ -15,6 +15,10 @@
 if TYPE_CHECKING:
     from ..function_graph import FunctionGraph
 
+    FromValueFunc = Callable[
+        [Any, (FunctionGraph, None), Tracker], "VariableBase"
+    ]
```

Comment:
```suggestion
    FromValueFunc = Callable[
        [Any, Optional[FunctionGraph], Tracker], "VariableBase"
    ]
```

另外在 TYPE_CHECKING 下面 `"VariableBase"` 还需要加引号嘛？

Reply:
这里我试了下 TYPE_CHECKING里也还是需要加上引号的

Reply:
唔，按理说运行时是不会跑这里的，那就是 type checker 的问题咯……

Reply:
感觉应该是的